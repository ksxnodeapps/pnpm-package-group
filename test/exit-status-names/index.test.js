'use strict'
const main = require('../../lib/exit-status-names')

describe('main module', () => {
  it('should stays unchanged', () => {
    expect(main).toMatchSnapshot()
  })
})
