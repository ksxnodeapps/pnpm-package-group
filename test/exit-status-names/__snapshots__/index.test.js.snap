// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main module should stays unchanged 1`] = `
Object {
  "enums": Object {
    "arguments": 2,
    "empty-input": 5,
    "filesystem": 6,
    "other": 1,
    "parsing-error": 3,
    "schema": 7,
    "stdin": 4,
    "success": 0,
  },
  "exit": [Function],
  "get": [Function],
  "names": Array [
    "success",
    "other",
    "arguments",
    "parsing-error",
    "stdin",
    "empty-input",
    "filesystem",
    "schema",
  ],
}
`;
